src/Http/Routing/src/ParameterPolicyActivator.cs

```diff
@@ -95,7 +95,7 @@ private static IParameterPolicy CreateParameterPolicy(IServiceProvider? serviceP
     {
         ConstructorInfo? activationConstructor = null;
         object?[]? parameters = null;
-        var constructors = parameterPolicyType.GetConstructors();
+        var constructors = parameterPolicyType.GetConstructors(BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Instance);
```

Comment:
```suggestion
        var constructors = parameterPolicyType.GetConstructors();
```
If this did change, then we'd also need to update the DAM attribute on the constraint map. Currently, it is `DynamicallyAccessedMemberTypes.PublicConstructors` only.